src/api/api.ts

```diff
@@ -0,0 +1,6 @@
+const API_URL = 'http://www.omdbapi.com/?apikey=443fd0d5&';
```

Comment:
```suggestion
const API_URL = 'https://www.omdbapi.com/?apikey=443fd0d5&';
```
Заміни тут URL, тому що на gh-pages не працює